views/js/pciCreator/ims/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -1038,9 +1074,17 @@ define([
          * @param {Object} [state] - the json serialized state object, returned by previous call to getStatus(), use to initialize an
          */
         getInstance: function getInstance(dom, config, state) {
+            const dispatchInteractiontraceEvent = detail => {
+                const interactiontraceEvent = new CustomEvent('interactiontrace', {
```

Comment:
**question:** Is the event name the expected one? In the ticket, I rather see `feedtrace`.

Reply:
As we discussed, it is not an issue. I've added a comment to the ticket.